src/shared/components/fadeInteraction/FadeInteraction.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import {assert} from 'chai';
+import { mount, shallow } from 'enzyme';
+
+import FadeInteraction from './FadeInteraction';
+
+
+describe('FadeInteraction',function(){
+
+    it('showByDefault prop sets initialShow state properly', function(){
+        const instance = shallow(<FadeInteraction showByDefault={true} />).instance() as FadeInteraction;
+        assert.isTrue(instance.initialShow, 'showByDefault results in initialshow=true');
```

Comment:
there must be a way to test this that actually tests the functionality? i.e. test if something is actually visible

Reply:
that is really something for the e2e tests.